common/models/src/auth/password.rs

```diff
@@ -0,0 +1,9 @@
+use crate::auth::AuthError;
+
+pub fn bcrypt_hash(password: &str) -> Result<String, AuthError> {
+    Ok(bcrypt::hash(password, bcrypt::DEFAULT_COST)?)
+}
+
+pub fn bcrypt_verify(password: &str, hash_password: &str) -> Result<bool, AuthError> {
+    Ok(bcrypt::verify(password, hash_password)?)
```

Comment:
Our code is open source. If someone looks at our auth code, can  they use our code to decipher the password as well？

Reply:
The hash function of bcrypt has the characteristics of one-way and randomness, so cracking bcrypt requires a lot of calculation and time, and it may even take decades, making it very difficult to crack bcrypt.